.github/workflows/stale-issue.yml

```diff
@@ -0,0 +1,55 @@
+# GitHub action based on https://github.com/actions/stale
+
+name: 'Close stale issues and PRs'
+on:
+  # !!! To be removed when merged to master
+  workflow_dispatch:
+  schedule:
+    # Execute every day 4:00 UTC
+    - cron: '0 4 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          # GLOBAL ------------------------------------------------------------
+          # !!! Debug is on until stabilized
+          debug-only: true
+          exempt-all-milestones: true
+
+          # ISSUES ------------------------------------------------------------
+          days-before-issue-stale: 365
+          stale-issue-label: stale
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has not had recent activity.
+            Given the limited bandwidth of the team, it will be automatically closed if no further
+            activity occurs. If you're interested in how we try to keep the backlog in a healthy state,
+            please read our [blog post on how we refine our backlog](https://blog.gradle.org/stale-issue-backlog).
+            If you feel this is something you could contribute, please have a look
+            at our [Contributor Guide](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md).
+            Thank you for your contribution.
+          days-before-issue-close: 21
+          close-issue-message: >
+            This issue has been automatically closed due to inactivity. If you can reproduce this on a
+            recent version of Gradle or if you have a good use case for this feature, please feel free
+            to to let know so we can reopen the issue. Please try to provide steps to reproduce, a quick
+            explanation of your use case or a high-quality pull request.
+          close-issue-reason: not_planned
+
+          # PULL REQUESTS -----------------------------------------------------
+          days-before-pr-stale: 60
+          stale-pr-label: stale
+          stale-pr-message: >
+            This pull request has been automatically marked as stale because it has not had recent activity.
+            Given the limited bandwidth of the team, it will be closed if no further activity occurs.
+            If you intend to work on this pull request, please ask the team to reopen the PR or push a new PR.
+            Thank you for your contributions.
+          days-before-pr-close: 21
+          close-pr-message: >
+            This issue has been automatically closed due to inactivity. If you can reproduce this on a
+            recent version of Gradle or if you have a good use case for this feature, please feel free
+            to to let know so we can reopen the issue. Please try to provide steps to reproduce, a quick
+            explanation of your use case or a high-quality pull request.
+          close-pr-reason: not_planned
```

Comment:
I don't think there is a `close-pr-reason`.

Reply:
Correct, blindly copied it, and didn't realize it doesn't exists.